### ISSUE TITLE
callback arguments

### ISSUE THREAD

**Issue by cjessett** [gyurcigyurma/Gather#1]
(Simplify) [Here](https://github.com/gyurcigyurma/Gather/blob/2e5956373e9cc489ce73ff07c027a7bbb02747ce/routes/index.js#L5) you should omit the `next` callback argument since it isn't used.

**Comment by gyurcigyurma** [comment]
Yes, thank you. On my local machine I installed eslint and got rid of these issues. I planned to do this, but when I submitted it was 5AM...
